src/lib/dfp/prepare-googletag.ts

```diff
@@ -111,11 +111,14 @@ export const init = (): Promise<void> => {
 			} else if (consentState.aus) {
 				// AUS mode
 				// canRun stays true, set NPA flag
```

Comment:
We can remove this comment

Reply:
We're still setting the flag, just in a different way but the comment does seem a bit redundant!